packages/cache-list/lib/AmpCaches.js

```diff
@@ -27,11 +27,10 @@ class AmpCaches {
   /**
    * Creates a new instance of AmpCaches.
    *
-   * @param {Object} [fetchStrategy = OneBehindFetch] The fetch strategy to be used when fetching
-   * data from the caches endpoint. Defaults to OneBehindFetch.
+   * @param {Function} fetch - a fetch implementation
    */
-  constructor(fetchStrategy = OneBehindFetch.create()) {
-    this.fetchStrategy_ = fetchStrategy;
+  constructor(fetch=oneBehindFetch) {
```

Comment:
nit: missing spaces around `=`: `fetch = oneBehindFetch`

Reply:
according to eslint says it's OK :-)